class-gwiz-gf-feed-forge.php

```diff
@@ -426,10 +426,16 @@ public function process_feeds() {
 	 * @return string The batch option name.
 	 */
 	public static function process_entry_feeds( $entries, $feeds, $form_id ) {
-		$addons = self::registered_addons();
-
+		$addons     = self::registered_addons();
 		$feed_cache = array();
 
+		/**
+		 * Filters whether to reprocess feeds that have already been processed.
+		 *
+		 *  @since 1.1
```

Comment:
```suggestion
		 * Filters whether to reprocess feeds that have already been processed.
		 *
		 * @param bool $reprocess_feeds Whether to reprocess feeds.
		 *
		 *  @since 1.0.1
```